addons/mail/static/tests/message/message_tests.js

```diff
@@ -1135,6 +1135,66 @@ QUnit.test("allow attachment image download on message", async () => {
     await contains(".o-mail-AttachmentImage .fa-download");
 });
 
+QUnit.test("check visibility of Download All button", async () => {
```

Comment:
Name of test is poor. Try as much as possible to name it so that functionally it's clear.
For example in this test, would be _"Can download all files of a message"_